frontend/src/hooks/api/integrations/types.ts

```diff
@@ -57,6 +57,10 @@ export type TIntegration = {
     shouldMaskSecrets?: boolean;
     shouldProtectSecrets?: boolean;
     shouldEnableDelete?: boolean;
+
+    awsIamRole?: string;
+    awsRegion?: string;
+    region?: string;
```

Comment:
Why both `awsRegion` and `region`? Can't we stick to one? (Preferably awsRegion)

Reply:
Other integrations may have a region, not sure which ones yet
so added explicit AWS for aws ones, and generic region for non aws